src/chat/chatter.py

```diff
@@ -273,43 +272,56 @@ def updatePlayer(self):
         # Rating icon choice  (chr(0xB1) = +-)
         formatting = ("Global Rating: {} ({} Games) [{}\xb1{}]\n"
                       "Ladder Rating: {} [{}\xb1{}]")
-        self.rankItem.setToolTip(formatting.format(
+        tooltip_str = formatting.format(
             str(int(rating)),
             str(player.number_of_games),
             str(int(player.rating_mean)),
             str(int(player.rating_deviation)),
             str(int(ladder_rating)),
             str(int(player.ladder_rating_mean)),
-            str(int(player.ladder_rating_deviation))))
+            str(int(player.ladder_rating_deviation)))
 
         league = player.league
         if league is not None:
-            formatting = ("Division : {}\n"
-                          "Global Rating: {}")
-            self.rankItem.setToolTip(formatting.format(
-                league["division"],
-                str(int(rating))))
-            self.rankItem.setIcon(util.THEME.icon("chat/rank/%s.png" % league["league"]))
+            icon_str, tooltip_str = league["league"], "Division : " + league["division"] + "\n" + tooltip_str
         else:
-            self.rankItem.setIcon(util.THEME.icon("chat/rank/newplayer.png"))
-
-        self._verifySortOrder()
+            icon_str = "newplayer"
+        self.rankItem.setIcon(util.THEME.icon("chat/rank/%s.png" % icon_str))
+        self.rankItem.setToolTip(tooltip_str)
 
     def updateGame(self):
         # Status icon handling
         game = self.user_game
-        player = self.user_player
         if game is not None and not game.closed():
-            url = game.url(player.id)
+            game_str = "<br/>title: " + game.title + "<br/>mod: " + game.featured_mod + "<br/>map: " + \
+                       maps.getDisplayName(game.mapname) + "<br/>players: " + \
+                       str(game.num_players) + " / " + str(game.max_players) + "<br/>id: " + str(game.uid)
             if game.state == GameState.OPEN:
-                self.statusItem.setIcon(util.THEME.icon("chat/status/lobby.png"))
-                self.statusItem.setToolTip("In Game Lobby<br/>"+url.toString())
+                if game.password_protected:
+                    game_str = " (private) Game Lobby:</b>" + game_str
+                else:
+                    game_str = " Game Lobby:</b>" + game_str
+                if game.host == self.user.name:
+                    icon_str, tooltip_str = "host", "<b>Hosting"
+                else:
+                    icon_str, tooltip_str = "lobby", "<b>In"
             elif game.state == GameState.PLAYING:
-                self.statusItem.setIcon(util.THEME.icon("chat/status/playing.png"))
-                self.statusItem.setToolTip("Playing Game<br/>"+url.toString())
-        else:
-            self.statusItem.setIcon(QtGui.QIcon())
-            self.statusItem.setToolTip("Idle")
+                if game.launched_at:
+                    delay = time.time() - game.launched_at + 1  # + 1 async jitter
```

Comment:
We shouldn't do it like this. If we want to know precisely if the game has a live replay already, we can add extra data to Game class that gets set before it emits the live replay signal and expose it as a property.

Reply:
oh, you're so right ... "game.has_live_replay"    was already there, how did I miss that ...